packages/app/src/app/pages/Dashboard/Content/routes/PathedSandboxes/Folders.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import FolderEntry from '../../../Sidebar/SandboxesItem/FolderEntry';
+import getChildCollections from '../../../utils/getChildCollections';
+
+import { Folder, FoldersWrapper } from './elements';
+
+const getPath = name => {
+  const slicedPath = window.location.pathname.split('sandboxes');
+  const last = slicedPath[slicedPath.length - 1];
+  return last && !last.includes('sandboxes')
+    ? `${decodeURIComponent(last)}/${name}`
+    : '/' + name;
+};
+
+const Folders = ({ loading, me, history, teamId }) => {
+  let folderInfo = {};
+  if (!loading) {
+    folderInfo = getChildCollections(
+      me.collections,
+      (me.collection || {}).path
+    );
+  }
+
+  const { children, foldersByPath, folders } = folderInfo;
+  return !loading && (children || []).size ? (
```

Comment:
I think it would be more readable to do an early return here:

```
if (loading) {
  return null;
}

const folderInfo = getChildCollections(
  me.collections,
  me.collection.path
);

if (folderInfo.children.size === 0) {
  return null;
}
```

This saves a lot of null checks.

Reply:
Good point! Changed :)